docs/source/samples.rst

```diff
@@ -0,0 +1,301 @@
+seen in the following two plots.
+
+.. plot:: :context: close-figs
+
+    fig, ax = plt.subplots(figsize=(5, 3))
+    ax = chain1.x0.plot.line(alpha=0.7, label="Chain 1")
+    ax = chain2.x0.plot.line(alpha=0.7, label="Chain 2")
+    ax.set_ylabel(chain1.get_label('x0'))
+    ax.set_xlabel("sample")
+    ax.legend()
+
+.. plot:: :context: close-figs
+
+    fig, axes = make_2d_axes(['x0', 'x1'], figsize=(5, 5))
+    chain1.plot_2d(axes, alpha=0.7, label="Chain 1")
+    chain2.plot_2d(axes, alpha=0.7, label="Chain 2")
+    axes.iloc[-1, 0].legend(bbox_to_anchor=(len(axes)/2, len(axes)), loc='lower center', ncol=2)
+
+
+Remove burn-in
+--------------
+
+To get rid of the initial burn-in phase, you can use the
+:meth:`anesthetic.samples.MCMCSamples.remove_burn_in` method:
+
+.. plot:: :context: close-figs
+
+    mcmc_burnout = mcmc_samples.remove_burn_in(burn_in=0.1)
+
+Positive ``burn_in`` values are interpreted as the *first* samples to
+*remove*, whereas negative ``burn_in`` values are interpreted as the *last*
+samples to *keep*. You can think of it in the usual python slicing mentality:
+``samples[burn_in:]``.
+
+If ``0 < abs(burn_in) < 1`` then it is interpreted as a fraction of the total
+number of samples in the respective chain.
+
+To see how ``remove_burn_in`` has gotten rid of the burn-in samples in both
+chains, see the plot in the following section, alongside an assessment of
+convergence.
+
+
+Gelman--Rubin statistic
+-----------------------
+
+Another important issue when it comes to MCMC samples is assessing convergence.
+In anesthetic we have implemented the modified Gelman--Rubin statistic as
+described in `Antony Lewis (2013) <https://arxiv.org/abs/1304.4473>`_. For the
+underlying, more theoretical accounts of this statistic, see e.g. `Gelman and
+Rubin (1992) <https://doi.org/10.1214/ss/1177011136>`_ and `Brooks and Gelman
+(1998) <https://doi.org/10.1080/10618600.1998.10474787>`_. 
+
+Provided you have an MCMC run containing multiple chains, you can compute the
+Gelman--Rubin ``R-1`` statistic using the
+:meth:`anesthetic.samples.MCMCSamples.Gelman_Rubin` method:
+
+.. plot:: :context: close-figs
+
+    Rminus1_old = mcmc_samples.Gelman_Rubin()
+    Rminus1_new = mcmc_burnout.Gelman_Rubin()
+
+The following plot shows how ``remove_burn_in`` gets rid of burn-in samples.
+Note the stark difference in the Gelman--Rubin statistic, as listed in the
+legend, depending on whether burn-in samples were removed or not.
+
+.. plot:: :context: close-figs
+
+    fig, axes = make_2d_axes(['x0', 'x1'], figsize=(5, 5))
+    mcmc_samples.plot_2d(axes, alpha=0.7, label="Before burn-in removal, $R-1=%.3f$" % Rminus1_old)
+    mcmc_burnout.plot_2d(axes, alpha=0.7, label="After burn-in removal,  $R-1=%.3f$" % Rminus1_new)
+    axes.iloc[-1, 0].legend(bbox_to_anchor=(len(axes)/2, len(axes)), loc='lower center')
+
+
+|
+
+Nested sampling statistics 
+==========================
```

Comment:
This page is fantastic, but it is a shame that it takes so long to get to some of the truly unique things that anesthetic offers. Could we flag this in the 'quickstart' since 'how do I get evidences out of my chains' is something that people would like to know (in addition to the more impressive plotting displayed there)

Reply:
There is the "Nested Sampling statistics" section in the `quickstart.rst`, does that not address this? 

I suppose the `quickstart.rst` could be fleshed out with some text, would that do the trick?

Reply:
Done in https://github.com/williamjameshandley/anesthetic/pull/243/commits/c1aac21be0390ed81ea75b501693bfad590e19d9.